e2e/dev/docs/.vitepress/config.ts

```
@@ -1,6 +1,7 @@
 import { fileURLToPath } from 'node:url'
 import { defineConfig } from 'vitepress'
 import { useSidebar } from 'vitepress-openapi'
+import { examplesPages, testsPages } from '../../../../docs/pages'
```

ESLint

[error] 'examplesPages' is defined but never used.

(unused-imports/no-unused-imports)